core/src/test/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupportTest.java

```diff
@@ -320,8 +323,9 @@ void requestTimedOutInServerSide() {
         final String serviceTag = "service=" + ctx.config().service().getClass().getName();
 
         final MeterIdPrefixFunction meterIdPrefixFunction = MeterIdPrefixFunction.ofDefault("foo");
+        final DistributionStatisticConfig distributionStatisticConfig = DEFAULT_DIST_STAT_CFG;
         RequestMetricSupport.setup(ctx, REQUEST_METRICS_SET, meterIdPrefixFunction, true,
-                                   SuccessFunction.ofDefault());
+                                   SuccessFunction.ofDefault(), distributionStatisticConfig);
```

Comment:
```suggestion
                                   SuccessFunction.ofDefault(), DEFAULT_DIST_STAT_CFG);
```
Please also fix the below changes as well. 😉